pages/index.js

```diff
@@ -25,8 +25,11 @@ export default function Home() {
             Our alliance comprises the following groups:<br></br>
             <b style={{color: '#358aca'}}>
               ✨ SWFL Coders
-              ✨ SWFL Data
+              ✨ SWFL Hackerspace
+              ✨ SWFL Tech Nights
               ✨ SWFL Sec
+            </br>
```

Comment:
Hi @aerikpawson thank you for giving this a shot 🙂  
The builds are erroring because this is React and React doesn't like to just add a closing tag. 
If you want ti break it into two lines you have to do it like we did on line 25.

Here is the build not passing for more info.
<img width="1195" alt="Screenshot 2024-03-25 at 12 29 57 AM" src="https://github.com/swfl-coders/tech-alliance-website/assets/15879194/a2531bc7-9f71-4686-b759-0d271d62c03e">

Reply:
@zarela My apologies for the noise! I clearly need to learn more about React and Vercel before pushing to the main repo.

Reply:
No worries! We are grateful you can help us 😃 
All you need to do is remove line 31